src/app/qgscrashhandler.cpp

```diff
@@ -59,7 +59,12 @@ LONG WINAPI QgsCrashHandler::handle( LPEXCEPTION_POINTERS exception )
   arguments = QCoreApplication::arguments();
   // TODO In future this needs to be moved out into a "session state" file because we can't trust this is valid in
   // a crash.
-  arguments << QgsProject::instance()->fileName();
+  QString projectFile = QgsProject::instance()->fileName();
+  if ( !projectFile.isEmpty() )
+    // quote project file path to avoid issues if it has spaces
+    arguments << '"' << projectFile << '"';
+  else
+    arguments << projectFile;
```

Comment:
shouldn't this be removed? it's appending an empty string?

Reply:
Oh, my bad. Fixed.

Reply:
Fixed.